File: JAVA/PetShop.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

class PetShop {
    private int id;
    private String namaProduk;
    private int hargaProduk;
    private int stokProduk;

    public PetShop() {
        this.id = 0;
        this.namaProduk = "";
        this.hargaProduk = 0;
        this.stokProduk = 0;
    }

    public PetShop(int id, String namaProduk, int hargaProduk, int stokProduk) {
        this.id = id;
        this.namaProduk = namaProduk;
        this.hargaProduk = hargaProduk;
        this.stokProduk = stokProduk;
    }

    public void setID(int id) { this.id = id; }
    public void setNamaProduk(String namaProduk) { this.namaProduk = namaProduk; }
    public void setHargaProduk(int hargaProduk) { this.hargaProduk = hargaProduk; }
    public void setStokProduk(int stokProduk) { this.stokProduk = stokProduk; }

    public int getID() { return this.id; }
    public String getNamaProduk() { return this.namaProduk; }
    public int getHargaProduk() { return this.hargaProduk; }
    public int getStokProduk() { return this.stokProduk; }
}

File: PHP/PetShop.php
<?php
require_once "PetShop.php";
class PetShop {
    protected $id;
    protected $nama_produk;
    protected $harga_produk;
    protected $stok_produk;
    protected $foto;

    public function __construct($id = 0, $nama_produk = "", $harga_produk = 0, $stok_produk = 0, $foto = "") {
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->harga_produk = $harga_produk;
        $this->stok_produk = $stok_produk;
        $this->foto = $foto;
    }

    public function setID($id) { $this->id = $id; }
    public function setNamaProduk($nama_produk) { $this->nama_produk = $nama_produk; }
    public function setHargaProduk($harga_produk) { $this->harga_produk = $harga_produk; }
    public function setStokProduk($stok_produk) { $this->stok_produk = $stok_produk; }
    public function setFoto($foto) { $this->foto = $foto; }

    public function getID() { return $this->id; }
    public function getNamaProduk() { return $this->nama_produk; }
    public function getHargaProduk() { return $this->harga_produk; }
    public function getStokProduk() { return $this->stok_produk; }
    public function getFoto() { return $this->foto; }
}

?>